src/processing/preprocessing.js

```diff
@@ -0,0 +1,17 @@
+const preprocess = (json) => {
+  const processed = json;
+
+  Object.keys(processed).forEach((key) => {
+    if (Array.isArray(processed[key]) || typeof processed[key] === 'object') {
+      processed[key] = preprocess(processed[key]);
+    } else if (Array.isArray(processed)) {
+      processed[key] = {
+        value: processed[key],
```

Comment:
Perhaps we should allow changing property name from `value` to something else via config file.